packages/manager/modules/anthos/src/dashboard/host/host-table/controller.js

```diff
@@ -0,0 +1,36 @@
+import Host from './host.class';
+import { STATUS } from './constants';
+
+export default class AnthosHostTableCtrl {
+  /* @ngInject */
+  constructor($translate, AnthosTenantsService) {
+    this.$translate = $translate;
+    this.AnthosTenantsService = AnthosTenantsService;
+    this.STATUS = STATUS;
+  }
+
+  loadHosts({ offset, pageSize }) {
+    this.error = null;
+    return this.AnthosTenantsService.getHosts(
+      this.serviceName,
+      this.additionalHosts,
+      (offset - 1) / pageSize + 1,
```

Comment:
👍

Reply:
Maybe a comment to explain why this is needed could be nice :)